jspsych-uil-session.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+if (typeof uil === 'undefined')
+    console.error("UiL Main library not loaded! " +
+                  "Refusing to load browser info functions");
+else
+{
+    if ('session' in uil) {
+        console.warn(
+            "uil.browser already exists, this is unexpected, " +
+                "we'll overwrite it."
+        );
+    }
+
+    uil.session = {};
+
+    (function (context) {
+        class API {
+            constructor(host, access_key) {
```

Comment:
This would be a better place for my earlier remark on the acces_key variable. Sorry, I'm not to comfortable with github reviews yet.